src/modules/delete.js

```diff
@@ -0,0 +1,16 @@
+// import UI from './func.js';
```

Comment:
- [ ] Please the following commented line of code too 😃 👍 this will help you keep your files clean 😉

Reply:
Thank you